<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetaDataWidget</class>
 <widget class="QWidget" name="MetaDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="upperArea" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="dateOfTestLabel">
        <property name="text">
         <string>Untersuchungsdatum</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="participantLineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="dateOfBirthLabel">
        <property name="text">
         <string>Geburtsdatum</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="participantLabel">
        <property name="text">
         <string>Name, Vorname</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="instructorLineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="instructorLabel">
        <property name="text">
         <string>Untersucher(in)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dateOfTestDateEdit"/>
      </item>
      <item row="2" column="3">
       <widget class="QDateEdit" name="dateOfBirthDateEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="remarksLabel">
     <property name="text">
      <string>Bemerkungen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="remarksPlainTextEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
